tag/mutations_test.go

```diff
@@ -0,0 +1,82 @@
+package tag_test
+
+import (
+	"testing"
+
+	"github.com/jinzhu/gorm"
+	"github.com/traggo/server/schema"
+	"github.com/traggo/server/tag"
+	"github.com/traggo/server/test"
+)
+
+func Test_createTag_succeeds(t *testing.T) {
+	query := `
+mutation {
+   createTag(key: "new tag", color: "#123456", type:novalue) {
+      key,
+      color,
+      type
+   }
+}`
+	expected := `
+{
+  "createTag": {
+    "color": "#123456",
+    "key": "new tag",
+    "type": "novalue"
+  }
+}`
+	test.NewGQL(t).Mutations(tag.Mutations).Exec(query).Succeeds(expected)
+}
+
+func Test_createTag_withExistingTag_fails(t *testing.T) {
+	createTag := func(db *gorm.DB) {
+		db.Create(schema.TagDefinition{Key: "existing tag", Color: "#fff", Type: schema.TypeSingleValue})
+	}
+	query := `
+mutation {
+   createTag(key: "existing tag", color: "#123456", type:novalue) {
+      key
+   }
+}`
+	expectError := "tag with key 'existing tag' does already exist"
+
+	test.NewGQL(t).Mutations(tag.Mutations).BeforeExec(createTag).Exec(query).Errs(expectError)
+}
+
+func Test_removeTag_succeeds(t *testing.T) {
+	createTag := func(db *gorm.DB) {
+		db.Create(schema.TagDefinition{Key: "existing tag", Color: "#fff", Type: schema.TypeSingleValue})
+	}
+	query := `
+mutation {
+   removeTag(key: "existing tag") {
+      key,
+      color,
+      type
+   }
```

Comment:
Where does this stand here for?
```
   {
      key,
      color,
      type
   }
```

For the first tests it kind of made sense, but here it confuses me.

Reply:
I this "defining" the return value?

Reply:
Wait... LOL ... that's just graphql ^^
So, yeah the "return value".